server/src/main/java/org/opensearch/action/search/SearchRequestContext.java

```diff
@@ -23,20 +23,19 @@
  */
 @InternalApi
 public class SearchRequestContext {
+    private final SearchRequest searchRequest;
```

Comment:
Could you please revert unrelated changes? They add a "noise" into review, thank you